src/submission/views.py

```diff
@@ -795,10 +794,11 @@ def proposal_data_processing(request, proposal, proposal_form_id):
             ]
 
     for field in data_fields:
-        if field.element.name in request.POST:
-            # TODO change value from string to list [value, value_type].
-            save_dict[field.element.name] = [
-                request.POST.get(field.element.name), 'text']
+        # Ensure ASCII field names.
+        field_name = field.element.name.encode('ascii', 'xmlcharrefreplace')
```

Comment:
You're doing this over and over throughout this process. Wouldn't it be useful to create a single method that does this, limiting the possibility of getting `.encode('ascii', 'xmlcharrefreplace')` wrong with any of the points it's used?

Reply:
Yeah fair enough!